config/default.ts

```diff
@@ -1,3 +1,7 @@
+import dotenv from "dotenv";
+
+dotenv.config({ path: "./.env" });
```

Comment:
This is not relevant to this PR. I just included it to load the env variables from the `.env` file so we don't have to add them every time.

Reply:
nice